backend/src/main/java/com/woowacourse/momo/group/event/GroupDeleteEvent.java

```diff
@@ -0,0 +1,16 @@
+package com.woowacourse.momo.group.event;
+
+import org.springframework.context.ApplicationEvent;
+
+import lombok.Getter;
+
+@Getter
+public class GroupDeleteEvent extends ApplicationEvent {
```

Comment:
```suggestion
public class GroupDeleteEvent {
```
Spring 4.2 부터 ApplicationEvent 상속 강제가 풀렸어요~!!와아아

https://docs.spring.io/spring-framework/docs/4.2.x/spring-framework-reference/htmlsingle/#context-functionality-events
<img width="90%" alt="스크린샷 2022-10-07 오전 11 28 18" src="https://user-images.githubusercontent.com/22176552/194453812-9bea0ee4-0107-46db-baa0-3db039433ab8.png">

ApplicationEventPublisher에 ApplicationEvent를 상속받지 않은 메서드를 지원하도록 오버로딩 메서드가 제공되었네요!!!
<img width="70%" src="https://user-images.githubusercontent.com/22176552/194453627-daf069b2-dc63-4390-b8c6-7f574a3afc98.png">

Reply:
Event는 보면 볼수록 새로운 기능이 많군요! ApplicationEvent의 상속 코드를 삭제했습니다!